docker/etna-base/entrypoints/build.sh

```diff
@@ -80,4 +80,4 @@ fi
 echo 'for file in /app/*.completion; do source $file || true; done' >> /root/.bashrc
 echo 'export PATH="/app/bin:$PATH"' >> /root/.bashrc
 # Allow other users to use the root bash setup
-chmod -R 777 /root/
+chmod 744 -R /root/
```

Comment:
I realized that I only needed this slightly lower permission grade so this change is pretty minor.